### ISSUE TITLE
Armor Bar displays very high numbers

### ISSUE THREAD

**Issue by DaddyP1G** [LordAnaku/Anaku_Status_Bars_Forge#1]
When you turn on the numbers for the bars, the armor bar shows a very high number.  
![2022-10-24_16 20 59](https://user-images.githubusercontent.com/37815251/197631801-acd357d6-58bd-406d-892d-07b1eb06a214.png)

**Comment by mihaaairu** [comment]
Yeah, have the same problem. + Armor bar doesn't expose any info about current armor. It always filled up because of high numbers.
